packages/addons-website/src/components/WebsiteSideNav/WebsiteSideNav-story.js

```diff
@@ -86,7 +86,7 @@ storiesOf('Website side nav', module)
             <SideNavLink
               icon={<Launch16 />}
               href="javascript:void(0)"
-              to="/resources#github"
+              to="/resources/#github-repos"
```

Comment:
Was only the light theme supposed to change ? I noticed the dark variation still says /#github

Reply:
ohh thats why the link was wrong on the homepage. yea we need to update both

Reply:
Ahhhh, thank you for catching that.